.gitignore

```
@@ -1,4 +1,5 @@
 /target
-
 # Generated by `oranda generate ci`
-public/
\ No newline at end of file
+public/
+# Generated by mdbook
+book/book/
```

And as long as I'm here
```suggestion
book/book/

```